### PR TITLE
Some minor edits to the amazing inference document.

### PR DESCRIPTION
**Pull request by steady137**
Would like to contribute, from humble beginnings. :)

### CODE REVIEW

File: source/documentation/inference/__index.md [hunk 1]
```diff
@@ -1054,8 +1054,8 @@ configuration still leaves something to be desired and will the subject of futur
 <p>We can create an instance of the OWL reasoner, specialized to the demo schema
   and then apply that to the demo data to obtain an inference model, as follows:</p>
 
-    Model schema = RDFDataMgr.loadModel("file:data/owlDemoSchema.owl");
-    Model data = RDFDataMgr.loadModel("file:data/owlDemoData.rdf");
+    Model schema = RDFDataMgr.loadModel("file:data/owlDemoSchema.xml");
+    Model data = RDFDataMgr.loadModel("file:data/owlDemoData.xml");
```
Comment (rvesse) [inline source/documentation/inference/__index.md:1058]: Might be better to rename the example data files to have `.rdf` extensions and correct the links in the above paragraph instead?

**Review by rvesse — APPROVED** [review]
Thanks for cleaning up, one minor comment and I'll request a review from our inference API expert as well before merging

File: source/documentation/inference/__index.md (hunk 1 above)
Comment (steady137) [reply source/documentation/inference/__index.md]: Sounds good. Please find it in the followup commit. I haven't generated the site to test it.

If you squash and rebase for PRs and I can push that instead. LMK?

**Review by der — APPROVED** [review]
Edits look to to me, many thanks.

**Pull request merged**